src/features/dashboard/components/ApiTokenTable/__tests__/index.test.tsx

```diff
@@ -72,76 +68,31 @@ describe('DeleteTokenDialog', () => {
     expect(token_row).toBeInTheDocument();
   });
 
-  it('Shows the dialog when pressing the delete button', async () => {
-    const delete_button = await screen.findByTestId('dt_delete_button_0');
-    expect(delete_button).toBeInTheDocument();
+  it('Should have the table headers and the header description', async () => {
+    const header_text = await screen.findByText(/API token manager/i);
+    expect(header_text).toBeInTheDocument();
 
-    await act(async () => {
-      await userEvent.click(delete_button);
-    });
-
-    const delete_modal = await screen.findByText(/Are you sure you want to delete this token?/i);
-    expect(delete_modal).toBeInTheDocument();
+    const header_descr = await screen.findByText(/Access all your API token details here/i);
+    expect(header_descr).toBeInTheDocument();
   });
 
-  it('Should close the dialog when pressing the close button', async () => {
-    const delete_button = await screen.findByTestId('dt_delete_button_0');
+  it('Shows the dialog when pressing the delete button', async () => {
+    const delete_button = await screen.findByTestId('delete-token-button');
+    await userEvent.click(delete_button);
     expect(delete_button).toBeInTheDocument();
```

Comment:
@agrim-deriv Expectation doesn't match with the test purpose here

Reply:
Combined this test purpose and the below one to combine it similar to how it is used in the apps table test case.